dspace-xmlui/src/main/java/org/dspace/app/xmlui/aspect/submission/submit/AccessStepUtil.java

```diff
@@ -188,31 +190,27 @@ else if (errorFlag == org.dspace.submit.step.AccessStep.STATUS_ERROR_MISSING_DAT
         }
     }
 
-    public void addEmbargoDateSimpleForm(DSpaceObject dso, List form, int errorFlag) throws SQLException, WingException {
+    public void addEmbargoDateDisplayOnly(final DSpaceObject dso, final List list) throws SQLException, WingException {
+        final Text text = list.addItem().addText("embargo");
+        text.setLabel(T_item_embargoed);
+        text.setHelp(T_label_date_displayonly_help);
+        populateEmbargoDetail(dso, text);
+        text.setDisabled(true);
+    }
 
-        String date=null;
-
-        if(dso!=null){
-            java.util.List<ResourcePolicy> policies = authorizeService.findPoliciesByDSOAndType(context, dso, ResourcePolicy.TYPE_CUSTOM);
-            if(policies.size() > 0){
-                ResourcePolicy rp = policies.get(0);
-                if(rp.getStartDate() != null)
-                {
-                    date = DateFormatUtils.format(rp.getStartDate(), "yyyy-MM-dd");
-                }
-                globalReason = rp.getRpDescription();
+    private void populateEmbargoDetail(final DSpaceObject dso, final Text text) throws SQLException, WingException {
+        final java.util.List<ResourcePolicy> policies = authorizeService.findPoliciesByDSOAndType(context, dso, ResourcePolicy.TYPE_CUSTOM);
```

Comment:
Back from vacation. Not sure why this didn't work for you, @tdonohue ; I'm giving it another spin myself. I believe I just kept it as TYPE_CUSTOM because that's the way it was before.